core/src/main/res/values/strings.xml

```diff
@@ -724,6 +724,8 @@
     <string name="add_podcast_by_url_hint" translatable="false">www.example.com/feed</string>
     <string name="browse_gpoddernet_label">Browse gpodder.net</string>
     <string name="discover">Discover</string>
+    <string name="discover_hide">Hide</string>
+    <string name="discover_will_be_hidden">You selected to hide podcasts using [%s].</string>
```

Comment:
I would simply use `You selected to hide suggestions.`. Having the text there again looks pretty technical.

Instead of `discover_will_be_hidden` I would use `discover_is_hidden`